src/lava/lib/peripherals/dvs/transform.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+from abc import abstractmethod
+from dataclasses import dataclass
+
+import numpy as np
+import copy
+import typing as ty
+
+
+@dataclass
+class EventVolume:
+    height: int
+    width: int
+    polarities: int
+    t_start: int = None
+    t_stop: int = None
+
+
+class Transformation:
```

Comment:
The class name and the file name should be the same. If we go with transform.py, this class should be called Transform for consistency.
Or in the other case, if we call this class Transformation, the file name should be transformation.py.